Xamarin.Forms.Platform.MacOS/Cells/CellRenderer.cs

```diff
@@ -45,7 +46,15 @@ protected void WireUpForceUpdateSizeRequested(ICellController cell, NSView nativ
 
 			_onForceUpdateSizeRequested = (sender, e) =>
 			{
-				//TODO: Implement ForceUpdateSize
+                var index = tableView?.RowForView(nativeCell);
+                if (index != null)
+                {
+                    NSAnimationContext.BeginGrouping();
```

Comment:
Why do we need this ?

Reply:
For NSView-based tables, this method will animate. To remove the animation, according to Apple's documentation, we need to set the duration to 0 in the current context.

https://developer.apple.com/documentation/appkit/nstableview/1532126-noteheightofrows